lib/developer_portal/app/views/developer_portal/shared/_swagger_ui.html.liquid

```diff
@@ -8,17 +8,79 @@
 </div>
 
 <script type="text/javascript">
+  var AUTH_METHOD_USER_KEY = 1;
+  var AUTH_METHOD_ID_KEY_PAIR = 2;
+
+  function getAuthParamsForMethod(authMethod) {
```

Comment:
https://github.com/3scale/porta/pull/539/files#r258106136
Is this method exactly the same? Then we should probably have it only once if possible 🤔